ui/app/styles/components/toggle.scss

```diff
@@ -31,7 +31,9 @@ $size: 12px;
   .toggler {
     display: inline-block;
     position: relative;
-    vertical-align: middle;
+    vertical-align: baseline;
+    position: relative;
+    top: 1px;
```

Comment:
This affects (positively) a few others instances where we use our <Toggle> element and it was slightly misaligned.